Packs/UrlScan/pack_metadata.json

```diff
@@ -2,7 +2,7 @@
     "name": "URLScan.io",
     "description": "urlscan.io Web Threat Intelligence",
     "support": "partner",
-    "currentVersion": "1.1.22",
+    "currentVersion": "1.2.0",
```

Comment:
This seems like a revision to me, are you sure you want it as minor change?

Reply:
Revision makes sense to me, I believe I have modified this to 1.1.23